### PR TITLE
Tracking doc: rewrite `@wdio/shared-store-service` package into TypeScript

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

rewrite `@wdio/shared-store-service` package into TypeScript #5845

## Types of changes

- [x] internal

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: packages/wdio-shared-store-service/src/server.ts [hunk 1]
```diff
@@ -1,7 +1,19 @@
-const polka = require('polka')
+// polka doesn't have up to date types
+// and the project is not maintained for 2 years
+const polka: Polka = require('polka')
 const { json } = require('@polka/parse')
```
Comment (christian-bromann) [inline packages/wdio-shared-store-service/src/server.ts:6]: can we use import here?

**Comment by mgrybyk** [comment]
ready for review

File: packages/wdio-shared-store-service/src/server.ts (hunk 1 above)
Comment (mgrybyk) [reply packages/wdio-shared-store-service/src/server.ts]: fixed

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**